flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/operations/SqlDdlToOperationConverterTest.java

```diff
@@ -2165,6 +2168,25 @@ public void testCreateViewWithDynamicTableOptions() {
         assertThat(operation).isInstanceOf(CreateViewOperation.class);
     }
 
+    @ParameterizedTest
```

Comment:
We could add the test at the syntax level since the operation level requires no change.